ivadomed/object_detection/utils.py

```diff
@@ -108,6 +106,7 @@ def generate_bounding_box_file(subject_path_list, model_path, path_output, gpu_i
         dict: Dictionary containing bounding boxes related to their image.
 
     """
+    from ivadomed import inference as imed_inference
```

Comment:
Here is a an example of what happens if this gets moved up top, causing circular import. 
![image](https://user-images.githubusercontent.com/6720068/123210717-68966d00-d490-11eb-8976-0f2628d16acc.png)

Reply:
@dyt811 I see that the circular import may be breaking the test. However, instead of  `from ivadomed import inference as imed_inference` why can we write `import ivadomed.inference as imed_inference` at the top.

From my understanding, what do you is not wrong but then is there a set way it has been done is the past. We can kep absolute import bound to a name at the top or move it later. Unless we don't have too many later imports, I don't see why what you have done can be a problem. If we do come across too many such (later) imports, it may affect code clarity.